SWLOR.Game.Server/Feature/GuiDefinition/PerksDefinition.cs

```diff
@@ -98,7 +98,7 @@ public GuiConstructedWindow BuildWindow()
                             });
                         })
                             .SetRowHeight(40f)
-                            .SetScrollbars(NuiScrollbars.Both)
+                            .SetScrollbars(NuiScrollbars.Y)
```

Comment:
What are the changes to the perk window here?

Reply:
An oopsie, I split these timers out from a larger PR and missed this file, mb